app/controllers/verifications_controller.rb

```diff
@@ -0,0 +1,33 @@
+class VerificationsController < ApplicationController
+  VERIFY_SID = Rails.application.credentials.twilio[:verify_sid]
+
+  def index
+  end
+
+  def send_code
+    $twilio.verify.services(VERIFY_SID).verifications.
+      create(to: twilio_phone_number, channel: 'sms')
+    flash[:notice] = 'Verification code has been sent'
+    redirect_to verify_path
+  end
+
+  def check_code
+    params.permit(:code)
+    code = params[:code]
+    check = $twilio.verify.services(VERIFY_SID).verification_checks.
+      create(to: twilio_phone_number, code: code)
+    if check.status == 'approved'
+      current_user.update(verified: true)
+      flash[:notice] = 'Phone number has been verified!'
+    else
+      flash[:notice] = 'Verification failed'
+    end
+    redirect_to verify_path
+  end
+
+  private
+
+  def twilio_phone_number
+    '+1' + current_user.phone.gsub(/\D/, '')
+  end
+end
```

Comment:
Curious, what did you run into that you needed to apply additional formatting to the phone number? Are we allowing invalid phone numbers into the database? Or is this for existing data that may not be valid?

We have a fairly strong validation setup here: https://github.com/OfficeMomsandDads/scheduler/blob/3cd893110079e1a4cd38d0760101e681b9802951/app/models/user.rb#L63

Reply:
The Twilio Verify API requires a strict phone format. 

> The phone number or email being verified. Phone numbers must be in E.164 format.

From https://www.twilio.com/docs/verify/api/verification

I tried sending it with my normal phone field and it didn't work. It has to start with "+1" and can't contain spaces, parentheses, or dashes. 
I checked and the standard Twilio Messages API also says it requires this, but it seems it doesn't? At least, our messages seem to be going through.

Reply:
Hmm okay, I guess our validation isn't as strict as I thought. Can you add a method on user that produces the phone number in `E.164` format?

Reply:
Sure, I'll move this controller method to user and also make a spec for it.

Reply:
Also the validation we have is fine, but it puts it in the more conventional user-readable (xxx) yyy-zzzz format, which Twilio doesn't like.